types/transformer/pathmapping.go

```diff
@@ -30,6 +30,8 @@ const (
 	ModifiedSourcePathMappingType PathMappingType = "SourceDiff" // Source path becomes relative to source directory
 	// PathTemplatePathMappingType allows for path template registration
 	PathTemplatePathMappingType PathMappingType = "PathTemplate" // Path Template type
+	// CustomTemplatePathMappingType allows copy of source to destination and applying of template with custom delimiter
+	CustomTemplatePathMappingType PathMappingType = "CustomTemplate" // Source path when relative, is relative to yaml file location
```

Comment:
Would Special be a better term here? Since it is not a customizable template, rather a special case to allow for specific usecases?

Reply:
done